src/shared/util.js

```diff
@@ -89,6 +89,7 @@ const AnnotationEditorParamsType = {
   HIGHLIGHT_COLOR: 31,
   HIGHLIGHT_DEFAULT_COLOR: 32,
   HIGHLIGHT_THICKNESS: 33,
+  HIGHLIGHT_HAS_THICKNESS: 34,
```

Comment:
Given how/where it's being used, would it perhaps make sense to just call it e.g. `HIGHLIGHT_FREE` instead?